examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala

```diff
@@ -46,10 +59,18 @@ object NaiveBayesExample {
 
     val predictionAndLabel = test.map(p => (model.predict(p.features), p.label))
     val accuracy = 1.0 * predictionAndLabel.filter(x => x._1 == x._2).count() / test.count()
+    println("model accuracy %f".format(accuracy))
```

Comment:
We have scala style rules that disable the use of println, you can check out some of the other examples in this directory to see how to disable them when we really want to use println (like here).